### ISSUE TITLE
Lack of organizing at meetings

### ISSUE THREAD

**Issue by TheDarkMuffin** [TheDarkMuffin/ft-reports#5]
Often, we'll get sidetracked and off topic.

Or people won't get to talk.

We need more rigor to our discussions.

**Comment by natashaaroberts** [comment]
I don't see getting "sidetracked" as a bad thing. Letting conversations be organic and flow naturally from one topic to the next is a positive. In addition, as we have a facebook group, anytime someone wishes to continue a topic, they can simply make a post online.

**Comment by TheDarkMuffin** [comment]
Yeah, but people actually have points. People like Bhavik like to prepare and it's unfair if they do that, and before they can get all the points they worked hard to bring to us, someone interrupts with a joke about how smelly their butt is and then we start talking about butts. Quite frankly, it's a bit rude.

I'd say natural evolution of the discussion would be fine, but I personally find people in our group to have poor judgement with when it's okay to say things. If someone spent all night preparing something for you and you just waltzed in, you kinda owe it to them to not interrupt, but a lot of people apparently don't feel that way. I'd like to give people the ability to exhaust a topic before we "naturally" move on.
